app/views/tenants/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<% @tenants.each do |tenant| %>
+  <p><%= tenant.organization %></p>
+<% end %>
```

Comment:
sweet, seems good to me. We can see all active and approved tenants on some page. Sweet.